README.md

```diff
@@ -18,10 +18,11 @@ Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
 
 ### Features
 
+* **Cloud-native registry**: Image management and helm charts management with similar user experiences are both supported. Harbor can serve cloud-native environments like container runtimes and orchestration platform [Kubernetes](https://kubernetes.io/) as cloud-native registry.
```

Comment:
Small rewrite, and also making sure we follow the [CNCF Style Guide](https://github.com/cncf/foundation/blob/master/style-guide.md) with regards to the spelling of "cloud native" (no hyphen).

```
**Cloud native registry**: With support for both container images and [Helm](https://helm.sh) charts, Harbor serves as registry for cloud native environments like container runtimes and orchestration platforms.
```